x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/analysis/ParsingTests.java

```diff
@@ -49,27 +49,27 @@ public class ParsingTests extends ESTestCase {
     );
 
     public void testCaseFunctionInvalidInputs() {
```

Comment:
All these changes here started to be "visible" thanks to https://github.com/elastic/elasticsearch/pull/117153. And now we see the "danger" of running two "sister" classes in parallel in two different modules ;-).